tabbycat/participants/templates/team_registration_card.html

```diff
@@ -10,17 +10,38 @@ <h4 class="card-title mb-0">
     </div>
 
       <div class="list-group-item">
-        {% if not use_code_names and team.short_name != team.long_name or private_url or admin_page %}
-          <strong>{% if participant %}{% trans "Team name:" %}{% else %}{% trans "Full name:" %}{% endif %}</strong>
-          {{ team.long_name }}
-            {% if not team.code_name or pref.team_code_names == 'off' %}
-              {% if pref.show_emoji %}({{ team.emoji }}){% endif %}
+        {% if not use_code_names or private_url or admin_page %}
+          {% if team.short_name == team.long_name %}
+            <div>
+              <strong>{% trans "Team name:" %}</strong>
+              {{ team.long_name }}
+            </div>
+          {% else %}
+            <div>
+              <strong>{% trans "Full team name:" %}</strong>
+              {{ team.long_name }}
+            </div>
+            <div>
+              <strong>{% trans "Short team name:" %}</strong>
+              {{ team.short_name }}
+            </div>
+          {% endif %}
+        {% endif %}
+        {% if pref.team_code_names != 'off' %}
+          <div>
+            <strong>{% trans "Code name:" %}</strong>
+            {% if team.code_name %}
+              {{ team.code_name }}
+            {% else %}
+              <span class="text-muted">{% trans "No code name assigned" %}</span>
             {% endif %}
+          </div>
         {% endif %}
-        {% if team.code_name and pref.team_code_names != 'off' %}
-          <br />
-          <strong>{% trans "Code name:" %}</strong>
-          {{ team.code_name }} {% if pref.show_emoji %}({{ team.emoji }}){% endif %}
+        {% if pref.show_emoji %}
+          <div>
+            <strong>{% trans "Emoji:" %}</strong>
+            {{ team.emoji }}
+          </div>
```

Comment:
Thought we were to append the emoji to the last displayed name rather than a new line for one character. Also, there might not be an emoji assigned.

Reply:
Yeah, I think at some point I came to the view that with all the conditional statements above it was too much effort to maintain that principle, lol. Definitely open to better ways to present this—I think the current layout isn't super easy to parse.

Good point about the possible null emoji, I added a line for that similar to code name.